code/modules/events/clownloose_spawn.dm

```diff
@@ -0,0 +1,70 @@
+/datum/round_event_control/clownloose
+	name = "Spawn Loose Clown"
+	typepath = /datum/round_event/ghost_role/clownloose
+	weight = 9999
+	max_occurrences = 1
+	min_players = 20
+	cannot_spawn_after_shuttlecall = TRUE
+
+/datum/round_event/ghost_role/clownloose/setup()
+	minimum_required = 1
+	role_name = ROLE_CLOWNLOOSE
+	fakeable = FALSE
+
+/datum/round_event/ghost_role/clownloose/spawn_role()
+	var/list/possible_spawns = list()
+	for(var/turf/L in GLOB.prisonspawn)
+		possible_spawns += L
+	if(!length(possible_spawns))
+		message_admins("No valid spawn locations found, aborting...")
+		return MAP_ERROR
+	var/turf/landing_turf = pick(possible_spawns)
+	var/list/candidates = get_candidates(ROLE_CLOWNLOOSE, /datum/role_preference/midround_ghost/clownloose)
+	var/result = spawn_clownloose(landing_turf, candidates, spawned_mobs)
+	if(result != SUCCESSFUL_SPAWN)
+		return result
+	priority_announce("CATCH THAT CLOWN!!! JAMES D. CENTCOM HERE, THAT FUCKING CLOWN ESCAPED FROM OUR PRISON AND STOLE MY HAT. HE JUST DROPPED in A DROP POD ON YOUR STATION. BRING HIM AND MY HAT BACK!! .", "Security Alert", SSstation.announcer.get_rand_report_sound())
```

Comment:
I think it would be funnier if it was formalised and vague akin to
"Delta level prisoner has breached centcom prisons and stolen a high priority officers belongings, be weary of any potential fugitives onboard"

Reply:
"Delta level prisoner has breached CentCom prisons and stolen a high rank officers headwear. Be weary of any potential fugitive onboard. Retrieve the prisoner and the stolen posetion back to CentCom."

Reply:
How is the return exactly done?

Reply:
Just

prevent the clown from getting it

Reply:
> I think it would be funnier if it was formalised and vague akin to "Delta level prisoner has breached centcom prisons and stolen a high priority officers belongings, be weary of any potential fugitives onboard"

This wording is worse, it's trying to take itself too seriously but "delta level" is too edgy and non-specific. It should mention their shuttle was seen in SS13's proximity unless it's a generic message that's sent to all the stations. It's more believable imo to have a clown as a low level pest that someone's really pissed about somehow